src/main/java/com/ss/ita/google/ui/pages/HomePage.java

```diff
@@ -38,5 +42,15 @@ private WebElement getLogo() {
     public boolean isLogoDisplayed() {
         return getLogo().isDisplayed();
     }
+
+    public HomePage hideLogo() {
+        setAttribute(getLogo(), "hidden", "true");
+        return this;
+    }
+
+    public HomePage displayLogo() {
+        removeAttribute(getLogo(), "hidden");
+        return this;
+    }
```

Comment:
re-do these into method: hideElement(element, true), where true - hide, false - show
move to util class